### ISSUE TITLE
Reading from incorrect codepage

### ISSUE THREAD

**Issue by jacob135evans** [pokepetter/ursina#81]
I'm having a strange issue which I can't find any documentation on. I used the example program:
```py
from ursina import *

app = Ursina()
ground = Entity(
    model = 'cube',
    color = color.magenta,
    z = -.1,
    y = -3,
    origin = (0, .5),
    scale = (50, 1, 10),
    collider = 'box'
)

app.run()
```
And in the log of the execution process I'm getting this:
```text
  File "C:\Users\jacob\AppData\Local\Programs\Python\Python38-32\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u0295' in position 32: character maps to <undefined>
```
This seems to be the issue however I don't know how to instruct the program to use my utf_8.py codepage because it is in that directory however it still uses cp1252.py

**Comment by pokepetter** [comment]
I think this is the bug with python 3.8.3. 
Version 3.8.4 and above should work.

**Comment by jacob135evans** [comment]
Thank you that is good to know!!

**Comment by jacob135evans** [comment]
I actually just got Python 3.8.4 installed and ran the same test, and it failed with the exact same result. I also tried Python 3.9.0 but Ursina wouldn't install via pip

**Comment by pokepetter** [comment]
Try a different terminal or use the newest ursina version from github. The terminal is failing to print a non-ascii character

**Comment by pokepetter** [comment]
Someone else had the issue with Atom's console. I don't know why it doesn't fall back to a box character like you'd expect.

**Comment by jacob135evans** [comment]
Ok, the issue has been resolved via the following process:

I was using the Atom package `atom-python-run  v0.9.7` from *foreshadow/atom-python-run*.
It allows for you to run Python code via the terminal using F5 and F6. So I set the F6 command to `chcp 65001 && python {file}` which runs `chcp 65001` to use the UTF-8 code page, if that succeeds it will run `python {file}` in this case `{file}` just means `{dir}/{name}{ext}` and it now runs properly.

Thanks again!
